sdk/core/Azure.Core.TestFramework/src/LiveOnlyAttribute.cs

```diff
@@ -11,7 +11,7 @@ namespace Azure.Core.TestFramework
     /// <summary>
     /// Attribute on test assemblies, classes, or methods that run only against live resources.
     /// </summary>
-    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class | AttributeTargets.Assembly, AllowMultiple = true, Inherited = true)]
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class | AttributeTargets.Assembly, AllowMultiple = false, Inherited = true)]
```

Comment:
This needed to be false to allow the LiveOnly attribute specified in derived classes to not be overwritten by the parent attribute.